examples/Clusters/03.k8scluster-managed-namespace.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: blocks/v1beta1
+kind: K8sCluster
+metadata:
+  name: default-cluster
+spec:
+  kubeConfig:
+    secretKeyRef:
+      # Supposing we have a service account Config stored in the secrets
+      # with access to only a specific namespace
+      key: kubeconfig
+      name: kubeconfig-default-cluster
+  # Pointing out a managedNamespace will make us
+  # deploy the helm releases on this namespace
```

Comment:
That's not entirely true. Remember that the ChartRelease decides in which namespace will the release be installed to. This makes sure that K8sCluster only manages this namespace.